From a0401ed86c9b1d8a4c7e0c4d990799e994b2a639 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Daniel=20Rodr=C3=ADguez=20Troiti=C3=B1o?=
 <danielrodriguez@fb.com>
Date: Tue, 16 Feb 2021 17:42:47 -0800
Subject: [PATCH 4/7] [android][tests] Add config.target_swift_api_extract for
 Android.

The configuration for Android of target_swift_api_extract was missing
and the CI machines could not start testing anything. These changes add
the missing line to the testing configuration.
---
 test/lit.cfg | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/test/lit.cfg b/test/lit.cfg
index f75db0126c6..d1f37aa64c3 100644
--- a/test/lit.cfg
+++ b/test/lit.cfg
@@ -1424,6 +1424,10 @@ elif run_os == 'linux-androideabi' or run_os == 'linux-android':
         config.swift_symbolgraph_extract,
         '-target', config.variant_triple,
         mcp_opt])
+    config.target_swift_api_extract = ' '.join([
+        config.swift_api_extract,
+        '-target', config.variant_triple,
+        '-sdk', shell_quote(config.variant_sdk)])
     config.target_swift_ide_test = ' '.join([
         config.swift_ide_test,
         '-target', config.variant_triple,
-- 
2.29.2

